source/getting-started/tutorials/timing-manager/index.md

```diff
@@ -0,0 +1,381 @@
+
+This tutorial provides code that allows users to experiment with the configuration of the AMDC [Timing Manager](/firmware/arch/timing-manager.md). The effects of this experimentation will be observed through the lens of the previous [Voltage Source Inverter](/getting-started/tutorials/vsi/index.md) tutorial.
+
+## Scheduling and Synchronizing
+
+Through the [Timing Manager](/firmware/arch/timing-manager.md), the AMDC is capable of synchronizing sensor data collection and task execution to the PWM carrier wave. This is desirable to eliminate electromagnetic noise in motor drives (for example, switching harmonics in the phase current measurements) because the inverter does not switch at the peak and trough of the PWM carrier.
+
+Additionally, updates to the PWM duty ratio can be precisely synced with the timing manager, and are by default.
+
+### Timing Manager Modes
+
+Every `X` PWM periods (where the user specifies `X` using the function `timing_manager_set_ratio()`), the AMDC will collect data from the sensors. By default, the [Timing Manager](/firmware/arch/timing-manager.md) is set to `Legacy Mode`, in which the AMDC OS will run tasks independent of whether the sensors have completed data collection.
+
+![](images/tmLegacySimple.svg)
+
+Alternatively, when configured in `Post-Sensor Mode`, the [Timing Manager](/firmware/arch/timing-manager.md) will instruct the AMDC OS to wait to execute tasks until the sensor data collection is complete. With proper configuration, this can be used to ensure the following:
+
+![](images/tmPostSensorSimple.svg)
+
+1. the task has new sensor data and
+2. the time elapsed between when the sensor data is acquired and when the task is run is consistent and jitter-free.
+3. the pwm duty ratio is updated at a consistent rate (every X pwm cycles)
+
+### AMDC Actions Synchronized by the Timing Manager
+
+There are multiple factors that affect when and how fast control tasks run:
+- User set `TASK_NAME_UPDATES_PER_SEC`
+- PWM Frequency
+- User Event Ratio
+- Sensor collection time (`Post-Sensor Mode`)
+- Control task time (how long it takes for the control task to run)
+
+Read [this docs page](/firmware/arch/timing-manager.md) for detailed information on the Timing Manager.
+
+Consider: Control tasks can run, at most, once every `X` PWM periods, where `X` is the User Event Ratio.\
+That means we have to satisfy these three inequalities:
+
+### Timing Manager Configuration Rules
+
+$$
+\frac{\rm PWM\_FREQUENCY}{\rm TASK\_NAME\_UPDATES\_PER\_SEC} \ge \rm EVENT\_RATIO \\
+\\
+\rm EVENT\_RATIO \ge {\rm PWM\_FREQUENCY} * {\rm Sensor\ Sample\ Acquisition\ Time} \\
+\\
+\rm Control\ Task\ Time > \frac{1}{\rm TASK\_NAME\_UPDATES\_PER\_SEC}
+$$ (eq:tm)
```

Comment:
Revise this so that each inequality has its own number (1), (2), (3) -- to make it easier to refer to each in the text below

Reply:
I tried to do this and couldn't figure out how with latex. I tried using {enumerate} to make a [numbered list](https://tex.stackexchange.com/questions/146420/making-a-numbered-list) but sphinx doesn't seem to have this feature.

Reply:
Nevermind, figured it out